tests/installation/first_boot.pm

```diff
@@ -22,19 +22,8 @@ sub handle_login {
     if (get_var('DM_NEEDS_USERNAME')) {
         type_string $username;
     }
-    if (match_has_tag("sddm")) {
-        # make sure choose plasma5 session
-        assert_and_click "sddm-sessions-list";
```

Comment:
these behavior to check the session is due to https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/81ef307d4ec6eeb85e32ed321b2f31ee355cdc16